packages/workbox-background-sync/src/lib/request-queue.js

```diff
@@ -117,7 +117,7 @@ class RequestQueue {
         id: hash,
         url: request.url,
       });
-    } catch(e) {
+    } catch (e) {
```

Comment:
Somewhat tangental, but in the future I'd love to see us standardize on `err` for `catch` blocks unless there's a good reason to use a more descriptive name. Maybe we could write an eslint plugin to enforce this...